CHANGELOG.md

```diff
@@ -102,6 +102,7 @@ Compatibility:
 * Show the pointer size information (if available) in `FFI::Pointer#inspect` (@nirvdrum).
 * Implement performance warnings (`Warning[:performance]`) like in CRuby 3.3 (@eregon).
 * The output of `Marshal.dump` is now compatible with CRuby for `Rational` and `Complex` instances (#3228, @eregon).
+* Support `connect_timeout` keyword argument to `TCPSocket.{new,open}` (#3421, @manefz, @patricklinpl, @nirvdrum, @rwstauner).
```

Comment:
it's probably a wrong release section (23.1.0)

Reply:
🤦